CSS3.sublime-syntax

```diff
@@ -7384,7 +7401,7 @@ contexts:
   property-margin-block-inline:
     - match: \b(margin-inline-start|margin-inline-end|margin-inline|margin-block-start|margin-block-end|margin-block)\s*(:)
       captures:
-        1: support.constant.property-name.css
```

Comment:
i changed these here.. can you help me understand why you'd chosen `constant` over `type`? i feel like constant is better for values than properties? i'd thought that things like `margin-inline` and `place-items` werent supported because they were rendering different than `justify-content`.

Reply:
Good question! There isn't much rationale behind the scope names. [The list of scope names](https://www.sublimetext.com/docs/3/scope_naming.html) comes from TextMate. Sublime used TextMate's syntax definition system for a long time, before finally devising a much better system with `sublime-syntax`. Unfortunately, the new system still uses the old scope names to avoid breaking all the existing color schemes. TextMate scope names were invented for C-like languages and don't map onto CSS well. So the choice of syntax names is pretty arbitrary.

I don't remember the reason I chose `constant` over `type`, but it was probably one of these:
1. I made a mistake.
2. I copied it from an old version of the official CSS syntax definition.
3. I liked the color of `constant` better.

I see that [the official CSS definition uses `type`](https://github.com/sublimehq/Packages/blob/5554fd25bcd43f642fa998a9f722f2cd43a6a254/CSS/CSS.sublime-syntax#L14080) now, so I'm definitely open to changing this. Why don't we hold off making that change in this PR and submit a followup PR to renames all instances of `support.constant.property-name`? Changing the scope names for just these properties might cause them to be different colors in some color schemes.

Reply:
can do